src/ResourceManagement/ResourceManager/Authentication/AzureCredentials.cs

```diff
@@ -146,20 +153,13 @@ public async override Task ProcessHttpRequestAsync(HttpRequestMessage request, C
                         throw new RestException($"Cannot communicate with server. ServicePrincipalLoginInformation should contain either a valid ClientSecret or Certificate information.");
                     }
                 }
-#if !NETSTANDARD
+#if (!NETSTANDARD && !NET461)
                 else if (userLoginInformation != null)
                 {
                     credentialsCache[adSettings.TokenAudience] = await UserTokenProvider.LoginSilentAsync(
                         userLoginInformation.ClientId, TenantId, userLoginInformation.UserName,
                         userLoginInformation.Password, adSettings, TokenCache.DefaultShared);
                 }
-#endif
-#if NETSTANDARD
```

Comment:
LoginByDeviceCodeAsync should be supported in .NetCore. Any particular reason to remove this functionality?

Reply:
The entire UserTokenProvider has been removed in package 2.3.3 for netstandard1.4